unlock-app/package.json

```diff
@@ -71,9 +61,7 @@
     "@svgr/cli": "^4.1.0",
     "depcheck": "^0.7.1",
     "nodemon": "^1.18.9",
-    "npm-check": "^5.9.0",
-    "react-router-test-context": "^0.1.0",
-    "web3-eth-abi": "1.0.0-beta.37"
+    "npm-check": "^5.9.0"
```

Comment:
can we move that to dev deps?

Reply:
it is in there ;) click the expand thing above!